Tutorial.md

```diff
@@ -1,16 +1,16 @@
 
-The workflow uses the Python programming language and a number of libraries developed at the Max-Planck-Institute for the Science of Human History (MPI-SHH), in particular the `lingpy` library. All these libraries are free and open source. For more information on LingPy, we recommend readers see the tutorial published as [supplementary material](https://github.com/lingpy/lingpy-tutorial) to "[Sequence comparison in computational historical linguistics](https://academic.oup.com/jole/article/3/2/130/5050100)" by List et al. (2018). 
+The workflow uses the Python programming language and several libraries developed at the Max-Planck-Institute for the Science of Human History (MPI-SHH), in particular the `lingpy` library. All these libraries are free and open source. For more information on LingPy, we recommend readers see the tutorial published as [supplementary material](https://github.com/lingpy/lingpy-tutorial) to "[Sequence comparison in computational historical linguistics](https://academic.oup.com/jole/article/3/2/130/5050100)" by List et al. (2018). 
 
 ## 1 Getting Started
 
 We assume readers are familiar with the essentials of command-line operation on either Unix-like systems (such as Linux and MacOS) or Windows systems. Likewise, we assume that the system is updated and that [Python](https://www.python.org/) is installed, along with its package manager `pip` and the `wheel` package. The latter might be necessary for installing the supporting libraries and, in those situations, it must be available *before* installing them.
 
-We strongly recommend using virtual environments for reproducing the steps outlined here. Virtual environments are a practical solution for creating independent configurations for testing and experimenting, with no interference on the system-wide installation. More information on virtual environments can be found on the [Python documentation](https://docs.python.org/3/tutorial/venv.html). In most systems, a virtual environment can be created by calling `python -m venv env`. Once created, the environment can be activated with `source env/bin/activate` and deactivated with the `deactivate` command.
+We strongly recommend using virtual environments for reproducing the steps outlined here. Virtual environments are a practical solution for creating independent configurations for testing and experimenting, with no interference on system-wide installation. More information on virtual environments can be found on the [Python documentation](https://docs.python.org/3/tutorial/venv.html). In most systems, a virtual environment can be created by calling `python -m venv env`. Once created, the environment can be activated with `source env/bin/activate` and deactivated with the `deactivate` command.
```

Comment:
"experimenting, with no interference on system-wide installation." -> "experimenting, that do not interfere with any other python programs installed on the system"